ingredient/ingredient-apim-api/src/functions.ts

```diff
@@ -21,4 +21,20 @@ export class ApimApiUtils extends BaseUtility {
 
         return backend;
     }
+
+    public get_hostheader(namespace: string, k8sHostname: string): string {
```

Comment:
should allow passing in the service name with a default of null, and using package shortName as backup.

Reply:
done

Reply:
added to the end to just keep optionals at the end - even though feels like it should go at the front